app/till_app.rb

```diff
@@ -0,0 +1,43 @@
+require 'sinatra/base'
+require './app/lib/till'
+
+class TillApp < Sinatra::Base
+
+  set :views, Proc.new { File.join(root, "views") }
+  set :public_folder, Proc.new { File.join(root, "public") }
+
+  till = Till.new
+
+  get '/' do
+    @menu_items = till.menu_read_items
+    @order_list = till.order_list
+    @receipt = till.print_receipt if till.receipt !=nil
+    @change = till.change
+    erb :index
+  end
+
+  post '/add_order_item?' do
```

Comment:
you can't add ? to urls! They have a specific meaning in http. Also you should make this RESTful - just call the route `post '/order_item'`